### DISCUSSION TITLE
TextEditor not display improvement?

### DISCUSSION THREAD

**Discussion by zukarusan** [AvaloniaUI/AvaloniaEdit#300] [Ideas]
Hi All!

I am recently into Avalonia/WPF then found this project.

But I encounter the common issue of [TextEditor not displaying](https://github.com/AvaloniaUI/AvaloniaEdit/issues/298) just because of style problem. It took me 2 hours of trials/errors and thankfully I found that github issue link

I think it's good to write a hint in Readme..? or maybe there's a workaround to include the style automatically?

Thanks

**Comment by danipen** [comment]
Agree. Yes feel free to add the details to the readme in a new PR
